src/week1/constants/identifier.ts

```diff
@@ -0,0 +1,7 @@
+const SERVICE_IDENTIFIER = {
+    MEMBER: Symbol.for("Member"),
+    MATCHER: Symbol.for("Matcher"),
+    DINNER: Symbol.for("Dinner")
+};
```

Comment:
```suggestion
const SERVICE_IDENTIFIER = {
    MATCHER: Symbol.for("Matcher"),
    DINNER: Symbol.for("Dinner")
};
```

예시) 이렇게 수정해보는건 어떨까요?

Reply:
```suggestion
const IDENTIFIER = {
    MATCHER: Symbol.for("Matcher"),
    DINNER: Symbol.for("Dinner")
}
```

이렇게 작성할 수도 있음

Reply:
```typescript
const IDENTIFIER = {
    MATCHER: Symbol.for("Matcher"),
    DINNER: Symbol.for("Dinner")
}
```